libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/GovCert.hs

```diff
@@ -48,14 +58,14 @@ govCertSpec ConwayGovCertEnv {..} vs =
               member_ key reps
           )
           -- ConwayAuthCommitteeHotKey
-          (branch $ \_ _ -> True)
+          (branch $ \coldCred _ -> ccCertSpec coldCred)
```

Comment:
If you want to make this test pass (rather than disable it), you have to change [this line above for the ConwayUnRegDRep branch] (https://github.com/IntersectMBO/cardano-ledger/pull/4436/files#diff-bf4f71ccf7c59b422f854e426f70ad6057608332a4c520149b48258f20030e1fR54)

to be just `True`

Reply:
Let me know if you want me to add a commit with this workaround.

Reply:
I just disabled it. We'll figure out what is wrong with it later